tests/test_route.py

```diff
@@ -50,13 +56,16 @@ def test_notification_route_response_unexpected_data(flask_test_client):
     """
     GIVEN: our service running on flask test client.
     WHEN: we post unexpected data to the endpoint "/send".
-    THEN: we expect a successful response 200.
+    THEN: we expect a unsuccessful response 400.
     """
 
     unexpected_data = {
-        "tpe": "MAGIC_LINK",
+        "te": "MAGIC_LINK",
```

Comment:
why did you change the field name?

Reply:
It is testing the incorrect key so made it a bit obvious that the key is incorrect 😄